Source/BackgroundCallNotifications.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  BackgroundCallNotifications.swift
+//  SparkSDK
+//
+//  Created by Rik van den Brule on 29/11/2016.
+//  Copyright © 2016 Cisco. All rights reserved.
+//
+
+import Foundation
+
+/// Defines notifications to notify SparkSDK of calls when the application is in the background.
+/// 
+/// These notifications can be posted to `NSNotificationCenter` to let SparkSDK handle background calls.
+@objc public class SparkBackgroundCallNotifications: NSObject {
```

Comment:
why not use following style?
extension Notification.Name {
    static let SparkCallIncomingInBackground = Notification.Name("com.ciscospark.SparkCallIncomingInBackground")
}